examples/official-site/sqlpage/sqlpage.yaml

```diff
@@ -2,4 +2,7 @@
 database_url: "sqlite::memory:"
 
 # We have a file upload example, and would like to limit the size of the uploaded files 
-max_uploaded_file_size: 256000
\ No newline at end of file
+max_uploaded_file_size: 256000
+
+# For the show-progress example to work, we have to disable compression
+compress_responses: False
```

Comment:
@lovasoa I'm not sure if this is acceptable for your website. The example works best live (I tried but failed to achieve the inline example style as for the 'form' component).

Reply:
nvm - I forgot I can disable compression with the header